### PR TITLE
Replace "sile" in the doc with "optuna".

### PR DESCRIPTION
**Pull request by sile**
Since there remains "sile" (the previous org name) in many places in the doc, this PR replaces it with "optuna".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@ kurobako
 
 [![kurobako](https://img.shields.io/crates/v/kurobako.svg)](https://crates.io/crates/kurobako)
 [![Documentation](https://docs.rs/kurobako/badge.svg)](https://docs.rs/kurobako)
-[![Actions Status](https://github.com/sile/kurobako/workflows/CI/badge.svg)](https://github.com/sile/kurobako/actions)
-[![Coverage Status](https://coveralls.io/repos/github/sile/kurobako/badge.svg?branch=master)](https://coveralls.io/github/sile/kurobako?branch=master)
+[![Actions Status](https://github.com/optuna/kurobako/workflows/CI/badge.svg)](https://github.com/optuna/kurobako/actions)
+[![Coverage Status](https://coveralls.io/repos/github/optuna/kurobako/badge.svg?branch=master)](https://coveralls.io/github/optuna/kurobako?branch=master)
 [![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE)
 [![Docker Hub](https://img.shields.io/badge/docker-ready-blue.svg)](https://hub.docker.com/r/sile/kurobako)
```
Comment (sile) [inline README.md:9]: Note that I keep the link untouched in this PR as the docker hub entry "optuna/kurobako" doesn't exist now.

**Comment by sile** [comment]
This PR also applies minor changes to resolve lint errors: https://github.com/optuna/kurobako/pull/39/commits/ee00f200ab22df59737d8976d4a7fe4e390e720d

File: kurobako_core/src/hypervolume.rs [hunk 2]
```diff
@@ -11,11 +11,11 @@ pub fn compute(pts: &[Vec<f64>], ref_pt: &[f64]) -> f64 {
 
 fn get_hypervolume_recursive(pts: &[Vec<f64>], ref_pt: &[f64]) -> f64 {
     match pts.len() {
-        1 => get_hypervolume_two_points(&pts[0], &ref_pt),
+        1 => get_hypervolume_two_points(&pts[0], ref_pt),
```
Comment (hvy) [inline kurobako_core/src/hypervolume.rs:14]: For the record, a needless borrow. https://github.com/optuna/kurobako/runs/2901652078?check_suite_focus=true#step:5:110

**Review by hvy — APPROVED** [review]
Is my understanding correct that the create registry will be updated with the next release? LGTM otherwise!

File: README.md (hunk 1 above)
Comment (hvy) [reply README.md:9]: Let's track it here https://github.com/optuna/kurobako/issues/40.

**Comment by sile** [comment]
> Is my understanding correct that the create registry will be updated with the next release? 

Yes.
And, I think we don't have to publish the next release immediately just to reflect this patch.

**Pull request merged**